### ISSUE TITLE
[Bug]: Can't find the file "\docs\public\*.png" please check it in "\docs\public\_meta.json".

### ISSUE THREAD

**Issue by iicdii** [web-infra-dev/rspress#1052]
### Version

```sh
System:
  OS: Windows 11 10.0.22631
  CPU: (16) x64 AMD Ryzen 7 7800X3D 8-Core Processor
  Memory: 17.10 GB / 31.15 GB
Browsers:
  Edge: Chromium (123.0.2420.97)
  Internet Explorer: 11.0.22621.1
npmPackages:
  rspress: 1.19.1 => 1.19.1
```

### Details

When upgrading the Rspress version from `1.19.0` to `1.19.1`, a warning occurs when running the dev command. Looking at the changelog, the change 'fix: filename contain extname like suffix by @Timeless0911 in [#1035](https://github.com/web-infra-dev/rspress/pull/1035)' seems to be most closely related to this warning.

```bash
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 1-19-46 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-15-45 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".  
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-16-47 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".  
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-21-40 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".  
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-21-47 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".  
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-21-9 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".   
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-22-29 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".  
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-57-30 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".  
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-7-28 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".   
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 2-8-20 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".   
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 3-0-55 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".   
warn    Can't find the file: C:\blog-rspress\docs\public\FEconf 2023 - TRACK A 3-1-10 screenshot.png, please check it in "C:\blog-rspress\docs\public\_meta.json".   
warn    Can't find the file: C:\blog-rspress\docs\public\blog_meta_image.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\chromium_emoji_syntax_error.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\css_background_example.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\css_background_example2.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\dark-logo.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\favicon.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\html_layout_only_div.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\html_layout_with_semantic.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\img_tag_example.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
warn    Can't find the file: C:\blog-rspress\docs\public\light-logo.png, please check it in "C:\blog-rspress\docs\public\_meta.json".
```

### Reproduce link

https://github.com/iicdii/blog-rspress

### Reproduce Steps

1. execute `pnpm install` && `pnpm remark build` && `pnpm dev`
2. find the warn message in console

**Comment by Timeless0911** [comment]
I will check it, thx~

**Comment by Timeless0911** [comment]
Fixed in v1.19.2
